Source/ACE.Entity/Enum/Properties/PropertyInt.cs

```diff
@@ -2,409 +2,409 @@
 {
     public enum PropertyInt : ushort
     {
-        Undef = 0,
-        ItemType = 1,
-        CreatureType = 2,
-        PaletteTemplate = 3,
-        ClothingPriority = 4,
-        EncumbranceVal = 5, // ENCUMB_VAL_INT,
-        ItemsCapacity = 6,
-        ContainersCapacity = 7,
-        Mass = 8,
-        ValidLocations = 9, // LOCATIONS_INT,
```

Comment:
We align the formatting on all of the other files - you want it unaligned? I am not sure I follow?